package.json

```diff
@@ -28,6 +28,7 @@
     "typed-error": "^3.1.0"
   },
   "devDependencies": {
+    "@types/node": "^12.0.2",
```

Comment:
This should be at most node 10 since that's the active LTS and what we normally use

Reply:
Ah, righto, it was automatically placed at 12.